### ISSUE TITLE
Documentation - add a section about `child` to Writing your first test

### ISSUE THREAD

**Issue by jenweber** [thefrontside/bigtest#815]
When I was trying to write some tests and interactors from scratch, I was a little confused about why and when to use `child`. I think this should have its own section on this page: https://frontside.com/bigtest/docs/platform/writing-your-first-test  I saw that I was not supposed to add steps after assertions, but it did not occur to me to use child.

Here are some reasons why someone someone should use `child`:
- Creating sets of steps and assertions
- Grouping interactions into logical sections that can be rearranged as the user flow changes

Here's why BigTest uses this structure syntax:
- BigTest uses an arrange/act/assert pattern
- It's fast to do tests this way

**Comment by jenweber** [comment]
@cowboyd do you have anything to add?

**Comment by cowboyd** [comment]
> @cowboyd do you have anything to add?

@jenweber yes, I would unpack the definition of "fast" to say that this allows us to optimize the test run without any extra effort by the user.  We already do some first order optimizations, but I foresee us being able to do things like transparently parallelize a test run across multiple browsers with zero changes to your test code.
